Makefile

```diff
@@ -138,6 +82,62 @@ $(error Can't build $(TARGET_ARCH) binaries on a $(HOST_ARCH) host)
 endif
 
 # ---------------------------------------------------------------------
+# Application configuration
```

Comment:
This snippet uses `TARGET_ARCH`, which is declared a bit above, so we needed to move down the whole section